src/backend/services/workflow_service.py

```
@@ -598,120 +798,168 @@ async def stream_execute_only(robot_code: str, user_query: str = None, workflow_
+        test_filename = "test.robot"
+        test_filepath = os.path.join(run_dir, test_filename)
 
-        # Execute test
-        logging.info(f"🚀 Executing test: {test_filename}")
-        result = run_test_in_container(client, run_id, test_filename)
-        yield f"data: {json.dumps({'stage': 'execution', **result})}\n\n"
+        try:
+            with open(test_filepath, 'w', encoding='utf-8') as f:
+                f.write(robot_code)
+            logging.info(f"📝 Saved test code to {test_filepath}")
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)